### ISSUE TITLE
TextField - screen reader ignores pre-existing error messages

### ISSUE THREAD

**Issue by phoinixi** [microsoft/fluentui#13309]
### Environment Information

- z**Package version(s)**: 7.89.0
- **Browser and OS versions**: Edge Version 83.0.478.37 and Windows 10 narrator

### Describe the issue:

When the user TAB on a `TextField` with an existing error message, the narrator will only say "invalid" but the error message will not be read. Although, when the user triggers an error, the message will be read as an alert.

### Please provide a reproduction of the issue in a codepen:

https://codepen.io/phoinixi/pen/GRpLWBa

#### Actual behavior:

The error message is read when the user triggers it but not if it is pre-existing on the page.

#### Expected behavior:

The error message should be always read.

**Comment by aneeshack4** [comment]
@phoinixi I think you might have to set the errorMessage to have a default, not necessarily behind the `if(!text)`... @ecraig12345 may have more insights into this.

**Comment by phoinixi** [comment]
@aneeshack4 I updated the codepen and it has now 2 inputs so it's easier to test:
if the error is triggered in both input, when switching from one input to another with keyboard, the screen reader will only say 'invalid', without reading the error message.

**Comment by OmarMustafa** [comment]
Hi @ecraig12345 can you please let me know if this is planned to be fixed any time soon, or if there's a work around?
This issue is causing A11ySev2 bugs to be filed against every team that uses <TextField />.
Thanks!

**Comment by vanessgo** [comment]
Gentle ping that this issue needs attention.

**Comment by dmp569** [comment]
It seems the issue might be related to how the error message is associated to the input control. The framework is injecting the error as an alert so that it is read immediately and automatically by the screen reader (which is good). You are also attaching an aria-describedby attribute to the <input> with the error (fine), however since the alert has been announced it seems like the screen reader stops announcing it since it feels it is done (and doesn't override the behavior even though the id attribute is valid for aria-describedby). 

Would it be possible to include an id attribute on one of the descendants of the <span> you use to hold the <div role="alert"> and then use that id as the aria-describedby value? I did some manual hack testing of this approach and it seemed to work fine.

**Comment by smhigley** [comment]
I narrowed this down to an Edge bug with how `aria-describedby` is mapped. They're not computing the description for the referenced node when it contains `role="status"` or `role="alert"`, and not setting the FullDescription prop in UIA.

Test cases here for reference, and a bug has been filed on Edge: https://jsfiddle.net/9dojch0m/

**Comment by spmonahan** [comment]
The Edge bug referenced above was fixed in Edge 98 (currently the Dev Channel release). I've tested with the example Codepen and the component works as expected.

I did have to make an update to the Codepen so it would build: https://codepen.io/seanms/pen/jOGZLjz?editors=0010

Closing as this issue is resolved pending the stable release of Edge 98.
